src/components/Contacts/Contacts.module.css

```diff
@@ -0,0 +1,3 @@
+.title {
+  text-align: center;
+} 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5750332)